src/main/resources/net/sf/jabref/gui/errorconsole/ErrorConsole.fxml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.control.Button?>
+<?import javafx.scene.control.ButtonBar?>
+<?import javafx.scene.control.DialogPane?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.control.ListView?>
+<?import javafx.scene.layout.BorderPane?>
+<DialogPane prefHeight="500.0" prefWidth="1080.0" xmlns="http://javafx.com/javafx/8.0.60" xmlns:fx="http://javafx.com/fxml/1" fx:controller="net.sf.jabref.gui.errorconsole.ErrorConsoleView">
+    <content>
+        <BorderPane>
+            <bottom>
+                <ButtonBar styleClass="custom-buttons" BorderPane.alignment="CENTER">
+                    <BorderPane.margin>
+                        <Insets top="10.0" />
+                    </BorderPane.margin>
+                    <buttons>
+                        <Button fx:id="closeButton" cancelButton="true" defaultButton="true" mnemonicParsing="false" onAction="#closeErrorDialog" text="%Close" textAlignment="CENTER" />
+                        <Button fx:id="copyLogButton" mnemonicParsing="false" onAction="#copyLogButton" text="%Copy Log" />
+                        <Button fx:id="createIssueButton" mnemonicParsing="false" onAction="#createIssueButton" text="%Report Issue" />
+                    </buttons>
+                </ButtonBar>
+            </bottom>
+            <center>
+                <ListView fx:id="allMessage" styleClass="list-content" BorderPane.alignment="CENTER" />
+            </center>
+        </BorderPane>
+    </content>
+   <header>
+      <Label fx:id="descriptionLabel" styleClass="info-section" text="%We now give you insight into the inner workings of JabRef's brain. These information might be helpful to diagnose the root cause of a problem. Please feel free to inform the developers about an issue." wrapText="true" />
```

Comment:
Do not use "JabRef's brain". Use "JabRef's internals".

Do not use "These information" use "This information".